### DISCUSSION TITLE
Scheduled backups

### DISCUSSION THREAD

**Discussion by Vertelleus** [emavgl/oinkoin#153] [Ideas]
With the 1.0.67 update I lost everything, even selecting to keep all data when uninstalled and downgrading to 1.0.66. Can you create a daily / week / month auto backup option? This will be the only way I would consider downloading this again.

**Comment by emavgl** [comment]
There is an open issue for that https://github.com/emavgl/oinkoin/issues/9

Did not find the time for that so far, but it is one of my highest priorities

**Comment by emavgl** [comment]
Hi,

I have a pull-request with the implementation of automatic backup + encryption. Unfortunately, it is not possible at the moment to select a destination folder.

Please give it a try :)

https://github.com/emavgl/oinkoin/pull/154#issuecomment-2351103520
